### PR TITLE
Update New-TransportRule.md

### PR DESCRIPTION
**Pull request by ybhargav1995**

### CODE REVIEW

File: exchange/exchange-ps/exchange/New-TransportRule.md [hunk 1]
```diff
@@ -2394,7 +2393,7 @@ The ExceptIfMessageTypeMatches parameter specifies an exception that looks for m
 
 - OOF: Auto-reply messages configured by the user.
 - AutoForward: Messages automatically forwarded to an alternative recipient.
-- Encrypted: Encrypted messages.
+- Encrypted: Encrypted messages. In thin clients like OWA (Outlook Web App) encryption as a message type is currently not supported.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/New-TransportRule.md:2396]:
```suggestion
- Encrypted: Encrypted messages. In thin clients like Outlook Web App (OWA), encryption as a message type is currently not supported.
```

**Review by JohanFreelancer9 — APPROVED** [review]

File: exchange/exchange-ps/exchange/New-TransportRule.md [hunk 2]
```diff
@@ -2394,7 +2393,7 @@ The ExceptIfMessageTypeMatches parameter specifies an exception that looks for m
 
 - OOF: Auto-reply messages configured by the user.
 - AutoForward: Messages automatically forwarded to an alternative recipient.
-- Encrypted: Encrypted messages.
+- Encrypted: Encrypted messages. In thin clients like Outlook Web App (OWA), encryption as a message type is currently not supported.
```
Comment (JohanFreelancer9) [inline exchange/exchange-ps/exchange/New-TransportRule.md:2396]:
```suggestion
- Encrypted: Encrypted messages. In thin clients like Outlook on the web, encryption as a message type is currently not supported.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Apologies @ybhargav1995, Outlook Web App has been rebranded. Could you please commit the suggested change? Many thanks.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by chrisda** [comment]
I made some edits:

- Unclear why AttachmentContainsWords was deleted from the syntax block, so I added it back. That parameter is definitely available in Exchange Online, and Exchange Server 2013-2019 in both New-TransportRule and Set-TransportRule.
- New-TransportRule and Set-TransportRule are basically identical, so anything you do in one needs to be done in the other.
- The description of a condition parameter is basically identical to the description of the corresponding exception parameter. So, whatever change you make to ExceptIfMessageTypeMatches, you also need to do to MessageTypeMatches.
- I noticed a strange problem for every parameter in the syntax block that contains []: there's a space between the parameter type and the [] that shouldn't be there (for example, `<RecipientIdParameter[]>` not `<RecipientIdParameter []>`). I'll fix that in a different pull request.

**Pull request merged**